Explain thermal storage facade in docs (#98)
oemof/oemof-thermal: docs/stratified_thermal_storage.rst

Usage
_____

StratifiedThermalStorage facade
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Using the StratifiedThermalStorage facade, you can instantiate a storage like this:

.. code-block:: python

  from oemof.solph import Bus
  from oemof.thermal.facades import StratifiedThermalStorage

  bus_heat = Bus('heat')

  thermal_storage = StratifiedThermalStorage(
      label='thermal_storage',
      bus=bus_heat,
      diameter=2,
      height=5,
      temp_h=95,
      temp_c=60,
      temp_env=10,
      u_value=u_value,
      min_storage_level=0.05,
      max_storage_level=0.95,
      capacity=1,
      efficiency=0.9,
      marginal_cost=0.0001
  )

The non-usable storage volume is represented by the parameters
:py:attr:`min_storage_level` and :py:attr:`max_storage_level`.

To learn about all parameters that can be passed to the facades, have a look at the
:ref:`api reference for the facade module <api_label>`.

> Storage investment is possible as well. To do so, you still need to provide :py:attr:`diameter`, but

Reviewer: ```suggestion
For the storage investment mode you need to provide :py:attr:`diameter`,
```

Reviewer: This is only a small remark, because you already mention above, that storage investment is possible.

Usage
_____

StratifiedThermalStorage facade
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Using the StratifiedThermalStorage facade, you can instantiate a storage like this:

Reviewer 2: ```suggestion
The storage is instantiated using the class StratifiedThermalStorage, which is part of oemof.thermal.facades modul:
```

Author: I intended to convey that using the facade is optional, therefore the formulation. Do you think it helps to mention that the StratifiedThermalStorage is located in the facades-module?

Reviewer 2: I see. So maybe an explanation in 'getting started' would be helpful at the end of the newly added paragraph? Something like:
For each technology the facade class can be found in the module oemof.thermal.facades.

Author: Yes, that sounds good! Can you write something up?

Reviewer 2: I've done this in commit https://github.com/oemof/oemof-thermal/pull/98/commits/09be1a619b434facae2b8f13fc75cc339cc3bdc0
